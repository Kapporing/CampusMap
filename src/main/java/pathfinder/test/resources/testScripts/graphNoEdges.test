# Create a graph with no edges
# attempts to get a path from two nodes
CreateGraph graph

AddNode graph node1
AddNode graph node2
AddNode graph node3

FindPath graph node1 node2
