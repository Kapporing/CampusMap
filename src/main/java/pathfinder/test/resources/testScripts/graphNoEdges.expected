# Create a graph with no edges
# attempts to get a path from two nodes
created graph graph

added node node1 to graph
added node node2 to graph
added node node3 to graph

path from node1 to node2:
no path found
